eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase.php

```diff
@@ -186,7 +186,7 @@ public function insertContentObject(CreateStruct $struct, $currentVersionNo = 1)
 
         $q->prepare()->execute();
 
-        return $this->dbHandler->lastInsertId(
+        return (int)$this->dbHandler->lastInsertId(
```

Comment:
those are type-casting bugs to be fixed for 7.5, unfortunately. Could be added to #2843.